x-pack/legacy/plugins/maps/public/layers/vector_layer.js

```diff
@@ -263,7 +271,7 @@ export class VectorLayer extends AbstractLayer {
       joinFields.push(...fields);
     });
 
-    return [...numberFieldOptions, ...joinFields];
+    return [...timeFieldOptions, ...numberFieldOptions, ...joinFields];
```

Comment:
Would this list time fields list before number fields in the UI? (Or is there something else sorting them later?) 🤔

Reply:
The list in the dropdown appears to be alphabetical by field label.